runatlantis.io/docs/server-configuration.md

```diff
@@ -927,6 +927,13 @@ and set `--autoplan-modules` to `false`.
   environment where releases.hashicorp.com is not available. Directory structure of the custom
   endpoint should match that of releases.hashicorp.com.
 
+### `--tf-disable-downloads`
```

Comment:
Instead of `--tf-disable-downloads` can we change it to something more intuitive
```suggestion
### `--download-tf`
```

```suggestion
### `--tf-download`
```

And we can set the above flag to default to `true` so if you want to disable it you can set is to `false`

Reply:
Fair enough about using positive wording 👍 
How about `--tf-allow-downloads`?

Reply:
Hm, at first I thought we can omit the word enable/disable and allow/deny from the wording since the env var would be a bit redundant ... ?

```
ATLANTIS_TF_ALLOW_DOWNLOADS=true
```

We really need to come together on a clear flag naming convention format.

cc: @runatlantis/maintainers @krrrr38 @Fabianoshz any recommendations on a flag name ?

Reply:
I know, this is a topic for an ADR in itself.

Reply:
At this point, I have no strong feelings about the naming 🤷 

Of the above suggestions, I've iterated using `--tf-download`, mainly because then its documentation naturally ends up beside the `--tf-download-url` documentation.

Reply:
sounds good to me!